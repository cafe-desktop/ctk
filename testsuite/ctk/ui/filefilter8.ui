<!-- test invalid file filter markup element -->
<interface>
  <object class="CtkFileFilter">
    <mime-types>
      <mime-type><warbl/></mime-type>
    </mime-types>
  </object>
</interface>
