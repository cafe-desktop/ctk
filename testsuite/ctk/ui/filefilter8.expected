ERROR: ctk-builder-error-quark 1
.:5:26 Unsupported tag for CtkFileFilter: <warbl>
